### DISCUSSION TITLE
Python version bump to 3.7.7?

### DISCUSSION THREAD

**Discussion by obradovic** [community/community#26996] [Actions]
<p>Hi there, it looks like python 3.7.6 was randomly deleted, in favor of 3.7.7.  We discovered this when our actions broke, saying:</p>
<p><span><span>##[error]Version 3.7.6 with arch x64 not found </span></span></p>
<p><span><span>Available versions: </span></span></p>
<p><span><span>2.7.18 (x64) </span></span></p>
<p><span><span>3.5.9 (x64) </span></span></p>
<p><span><span>3.6.10 (x64) </span></span></p>
<p><span><span>3.7.7 (x64) </span></span></p>
<p><span><span>3.8.2 (x64) </span></span></p>
<p>We sighed heavily and upgraded to 3.7.7. Unfortunately, we then got the OPPOSITE error message:</p>
<p><span><span>##[error]Version 3.7.7 with arch x64 not found </span></span></p>
<p><span><span>Available versions: </span></span></p>
<p><span><span>2.7.17 (x64) </span></span></p>
<p><span><span>3.5.9 (x64) </span></span></p>
<p><span><span>3.6.10 (x64) </span></span></p>
<p>3.7.6 (x64)<br>
3.8.2 (x64) </p>
<p>Could you please tell us what the heck?  Also, is there a way we can get pre-emptively notified when python versions disappear? Or perhaps make them not disappear?</p>

**Comment by Crow-EH** [comment, marked as answer]
<p>Hello <a class="mention" href="/obradovic">@obradovic</a>,</p>
<p>Are you using actions/setup-python ?<br>
If you are, <a href="https://github.com/actions/setup-python/issues/64#issuecomment-594837888" rel="nofollow noopener">according to an answer on a similar issue</a>, only the latest minor version is provided on runners.</p>
<p>You have two options: Either use 3.7 if you don’t mind about the minor version changing, or consider using the new python-versions feature through actions/setup-python@v2 ( <strong>available in v2 only</strong> ).</p>
<p>This feature will allow you to specify any version, if the major version is supported, and it will download it if it’s not already provided (<a href="https://github.com/actions/setup-python" rel="nofollow noopener">see the readme for more informations</a>).</p>
<p>You just have to upgrade to v2 and it should work out of the box. If you’re already using v2, well… I guess you can open an issue on their repository.</p>
<p>Hope that helped.</p>
